compiler/record-minmax-conversion-test/gen_h5_inputs.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import h5py as h5
+import numpy as np
+import tensorflow as tf
+import argparse
+
+#
+# This script generates a pack of random input data (.h5) expected by the input tflite model
+#
+# Basic usage:
+#   gen_h5_inputs.py --model <path/to/tflite/model> --num_data <number/of/data>
+#   ex: gen_h5_inputs.py --model add.tflite --num_data 3
+#   (This will create add.tflite.input.h5 composed of three random inputs in the same directory as the model)
+parser = argparse.ArgumentParser()
+parser.add_argument('--model', type=str, required=True)
+parser.add_argument('--num_data', type=int, required=True)
+args = parser.parse_args()
+
+model = args.model
+
+num_data = args.num_data
+
+# Build TFLite interpreter. (to get the information of model input)
+interpreter = tf.lite.Interpreter(model)
+interpreter.allocate_tensors()
+input_details = interpreter.get_input_details()
```

Comment:
(Optional) I think it would be even better if we can generate input data with `.circle` file using our interpreter or sth someday as well.